jerry-core/debugger/debugger.h

```diff
@@ -288,27 +296,23 @@ typedef struct
   uint8_t eval_size[sizeof (uint32_t)]; /**< total size of the message */
 } jerry_debugger_receive_eval_first_t;
 
-/**
- * Incoming message: next message of evaluating expression.
- */
-typedef struct
-{
-  uint8_t type; /**< type of the message */
-} jerry_debugger_receive_eval_part_t;
 
 /**
- * Data for evaluating expressions
+ * Incoming message: first message of client source.
  */
 typedef struct
 {
-  uint32_t eval_size; /**< total size of the eval string */
-  uint32_t eval_offset; /**< current offset in the eval string */
-} jerry_debugger_eval_data_t;
+  uint8_t type; /**< type of the message */
+  uint8_t code_size[sizeof (uint32_t)]; /**< total size of the message */
+} jerry_debugger_receive_client_source_first_t;
 
 void jerry_debugger_free_unreferenced_byte_code (void);
 
+void jerry_debugger_sleep (unsigned milliseconds);
```

Comment:
Since we always pass `JERRY_DEBUGGER_TIMEOUT`, this function could work without argument.